IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,91 @@
 
+enum Size: Double {
+    case small = 2.40
+    case medium = 3.50
+    case large = 5.00
+}
+
+struct Flavor {
+    var name: String
+    var rating: Float
+    
+    init(name: String, rating: Float) {
+        self.name = name
+        self.rating = rating
+    }
+}
+
+struct Cone {
+    var flavor: String
+    var topping: String?
+    var size: Size
+    
+
+    func eat() {
+        print("chocolate")
+    }
+}
+
+
+class IceCreamShop {
+    var toppings: [String]
+    var sizes: [Size]
+    var flavors: [Flavor]
+    var totalSales: Double = Double()
+    
+    
+    init() {
+        self.toppings = ["Caramel", "Green tea", "Rocky road"]
+        self.sizes = [.small, .medium, .large]
+        self.flavors = [Flavor(name: "Chocolate", rating: 4.0), Flavor(name: "Pinacolada", rating: 5.0), Flavor(name: "Butter Pecan", rating: 6.0), Flavor(name: "Vanilla", rating: 2.0), Flavor(name: "Strawberry", rating: 3.0)]
```

Comment:
While this initializer will technically work, you want to try and make your inits (and all your code really) are dynamic allowing you to pass in the different values as needed. This way will always initialize an ice cream shop with those toppings and flavors.